### PR TITLE
misc bessctl/protobuf fixes

### PR DESCRIPTION
**Pull request by melvinw**
Addressing

- Change PMDPortArg proto to only take oneof {port_id, pci, vdev}
- “monitor tc” numbers pkts/sched and cycles/p always end with “.000”
- Tab completion for certain command breaks bessctl when bessd isn’t running

### CODE REVIEW

**Comment by melvinw** [comment]
small request: don't squash these on merge. the change sets are somewhat disjoint

File: core/drivers/pmd.cc [hunk 1]
```diff
@@ -110,86 +110,92 @@ void PMDPort::InitDriver() {
...
-        if (ret < 0) {
-          return pb_error(ENODEV,
-                          "Cannot attach PCI "
-                          "device %s",
-                          name);
-        }
-
-        *ret_hot_plugged = true;
-      }
+static pb_error_t find_dpdk_port_by_id(dpdk_port_t port_id,
```
Comment (barath) [inline core/drivers/pmd.cc]: Could you add a function comment saying what the arguments are?

File: core/drivers/pmd.cc [hunk 2]
```diff
@@ -110,86 +110,92 @@ void PMDPort::InitDriver() {
...
+  }
+  if (!rte_eth_devices[port_id].attached) {
+    return pb_error(ENODEV, "Port id %d is not available", port_id);
+  }
+
+  *ret_port_id = port_id;
+  return pb_errno(0);
+}
+
+static pb_error_t find_dpdk_port_by_pci_addr(const std::string &pci,
```
Comment (barath) [inline core/drivers/pmd.cc]: Function comment.

File: core/drivers/pmd.cc [hunk 3]
```diff
@@ -110,86 +110,92 @@ void PMDPort::InitDriver() {
...
 
-  if (port_id == DPDK_PORT_UNKNOWN) {
-    return pb_error(EINVAL,
-                    "'port_id', 'pci', or 'vdev' field "
-                    "must be specified");
+  *ret_port_id = port_id;
+  return pb_errno(0);
+}
+
+static pb_error_t find_dpdk_vdev(const std::string &vdev,
```
Comment (barath) [inline core/drivers/pmd.cc]: Function comment

**Pull request merged**